### DISCUSSION TITLE
Physician Notes

### DISCUSSION THREAD

**Discussion by albu89** [MIT-LCP/mimic-code#1108] [MIMIC-IV]
Dear all, is my understanding correct that for research that requires the physician's notes we are advised to use the MIMIC III dataset? Are there any plans to release the physician notes for the MIMIC IV dataset? Thank you for a brief heads-up. Yours, Alex

**Comment by tompollard** [comment]
@albu89 that's correct, for now you would need to use MIMIC-III if the notes are required. We plan to release notes for MIMIC-IV in the future, but I'm not able to commit to a date (apologies!).

**Reply by AshCher51** [reply]
@tompollard wondering if there is any update from the MIMIC team regarding when notes may become available in MIMIC-IV?

**Reply by tompollard** [reply]
The dataset has been created and it is undergoing review by the editorial team at [PhysioNet](https://physionet.org/). It's difficult to say how long the review process will take, so unfortunately I'm not able to give an accurate time estimate.
